ideascube/blog/views.py

```diff
@@ -64,6 +64,7 @@ class ContentCreate(CreateView):
     form_class = ContentForm
 
     def get_initial(self):
+        self.object = Content(author = self.request.user)
```

Comment:
In fact, this is actually a problem, because it changes the `New post` title into `Edit post ""` when creating a new blog article.